### ISSUE TITLE
Important: Outgoing transactions not work

### ISSUE THREAD

**Issue by riartem** [blockchain/service-my-wallet-v3#261]
*Service v0.26.0, Node v8.4.0*
method /merchant/:guid/payment
return "error":"Unexpected error, please try again"
In JSON-RPC mode method *sendtoaddress* returned *Failed to build transaction*

Wallet was created with API. If I login to web wallet with same credentials — in it sending works.

**Comment by munkh-erdene** [comment]
Same here.

**Comment by 9xcoder** [comment]
Same!, nobody care this issue ??

**Comment by riartem** [comment]
@jtormey Are you seen that issue?

**Comment by shakilmia** [comment]
same tody

**Comment by BitEx21** [comment]
@riartem did you manage to get it working? getting same issue

**Comment by riartem** [comment]
@BitEx21 No, it's a server-side issue and can't be managed from our side.
I migrated to Blocktrail API and after little bit time my project was been closed.

**Comment by thongtd** [comment]
Same issue. How can I fix. Thanks
